astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -536,6 +536,67 @@ def divide(self, operand, operand2=None, **kwargs):
         return self._prepare_then_do_arithmetic(np.true_divide, operand,
                                                 operand2, **kwargs)
 
+    def pow(self, exponent, propagate_uncertainties=True, **kwargs):
+        """Raises ``self`` to the power of an exponent.
+
+        Non-integer and non-positive exponents are not supported
+        if uncertainties must be propagated.
+        **kwargs are passed to NDData constructor.
```

Comment:
I would move the description of `kwargs` to Parameters. Also, please wrap it in double-backticks if you keep the `**` there because otherwise Sphinx thinks you have started, but not ended, a block of bold text.